src/lib/types/experimental.ts

```diff
@@ -252,6 +253,10 @@ const flags: IFlags = {
         process.env.UNLEASH_EXPERIMENTAL_SCIM_API,
         false,
     ),
+    displayEdgeBanner: parseEnvVarBoolean(
+        process.env.UNLEASH_EXPERIMENTAL_DISABLE_SCHEDULED_CACHES,
```

Comment:
```suggestion
        process.env.UNLEASH_EXPERIMENTAL_DISPLAY_EDGE_BANNER,
```

although I don't think we need the env variable for this feature in particular